app.js

```diff
@@ -0,0 +1,110 @@
+// JS for booklist
+
+// Functions for local storage
+function getBooks() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+}
+
+function setBooks(books) {
+  if (books.length > 0) {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+function removeBook(index) {
+  const books = getBooks();
+
+  books.splice(index, 1);
+
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+// UI Class: Handle UI Tasks
+function addBookToList(book) {
+  const list = document.querySelector('#book-list');
+
+  const item = document.createElement('li');
+  item.className = 'd-flex flex-column border-bottom border-2 py-2';
+
+  const pBook1 = document.createElement('p');
+  pBook1.textContent = book.title;
+  pBook1.className = 'py-1 m-0';
+
+  const pBook2 = document.createElement('p');
+  pBook2.textContent = book.author;
+  pBook2.className = 'py-1 m-0';
+
+  const pRmv = document.createElement('button');
+  pRmv.textContent = 'Remove';
+  pRmv.className = 'py-1 m-0 rmv';
+
+  item.appendChild(pBook1);
+  item.appendChild(pBook2);
+  item.appendChild(pRmv);
+
+  list.appendChild(item);
+}
+
+function displayBooks() {
+  // Gets booklist from local storage
+  const books = getBooks();
+  books.forEach((book) => addBookToList(book));
+}
+
+
+// Event: Display Books
+document.addEventListener('DOMContentLoaded', displayBooks());
+
+// Event: Add book
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+
+  // Get form values
+  const titleI = document.querySelector('#book-title').value;
+  const authorI = document.querySelector('#book-author').value;
+
+  // Validate
+  if (titleI === '' || authorI === '') {
+    console.log('Error: title and author must not be empty');
```

Comment:
- [ ] I would recommend removing the `console.log` statement from the code. Since the user won't go to the browser console to check what happens, it's just for the developer debugging purpose.